### PR TITLE
Dualingo screen in event story screen

### PR DESCRIPTION
**Pull request by gull192**
Dualingo added to story screen

### CODE REVIEW

File: tv-app/effecticeTV/app/src/main/java/band/effective/office/tv/screen/duolingo/DuolingoScreen.kt [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    ) {
+        Column(modifier = Modifier
+                .fillMaxSize()
+                .padding(horizontal = 80.dp)
+        ) {
+            DuolingoTitle(Modifier.padding(start = 10.dp))
+            Spacer(modifier = Modifier.height(10.dp))
+            val textTitle = when (keySort){
+                KeySortDuolingoUser.Xp -> {
+                    "Tоп по XP"
```
Comment (Radch-enko) [inline tv-app/effecticeTV/app/src/main/java/band/effective/office/tv/screen/duolingo/DuolingoScreen.kt:36]: strings.xml

**Review by Radch-enko — APPROVED** [review]

File: tv-app/effecticeTV/app/src/main/java/band/effective/office/tv/core/ui/screen_with_controls/TimerSlideShow.kt
Comment (UserNameMax) [inline tv-app/effecticeTV/app/src/main/java/band/effective/office/tv/core/ui/screen_with_controls/TimerSlideShow.kt:1]: what you do with TODO

Comment (gull192) [reply tv-app/effecticeTV/app/src/main/java/band/effective/office/tv/core/ui/screen_with_controls/TimerSlideShow.kt:1]: this TODO needs to be fixed in another branch

File: tv-app/effecticeTV/app/src/main/java/band/effective/office/tv/network/use_cases/EventStoryDataUseCase.kt [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+            .toUI()
+
+    private fun setDuolingoDataForScreens(duolingoUsers: List<DuolingoUser>) =
+        run {
+            val userXpSort = DuolingoUserInfo(
+                users = duolingoUsers
+                    .sortedByDescending { it.totalXp }
+                    .subListForScreen(),
+                keySort = KeySortDuolingoUser.Xp
+            ) as StoryModel
```
Comment (UserNameMax) [inline tv-app/effecticeTV/app/src/main/java/band/effective/office/tv/network/use_cases/EventStoryDataUseCase.kt:65]: Why need this cast? Do StoryModel is super class for DualingoUserInfo or not?

File: tv-app/effecticeTV/app/src/main/java/band/effective/office/tv/network/use_cases/EventStoryDataUseCase.kt [hunk 3]
```diff
@@ -0,0 +1,75 @@
+package band.effective.office.tv.network.use_cases
```
Comment (UserNameMax) [inline tv-app/effecticeTV/app/src/main/java/band/effective/office/tv/network/use_cases/EventStoryDataUseCase.kt:1]: use case is part of domain

File: tv-app/effecticeTV/app/src/main/java/band/effective/office/tv/network/use_cases/EventStoryDataUseCase.kt (hunk 2 above)
Comment (gull192) [reply tv-app/effecticeTV/app/src/main/java/band/effective/office/tv/network/use_cases/EventStoryDataUseCase.kt:65]: Yes, but it's more readable in my opinion

File: tv-app/effecticeTV/app/src/main/java/band/effective/office/tv/screen/duolingo/utils/FlagMapper.kt
Comment (UserNameMax) [inline tv-app/effecticeTV/app/src/main/java/band/effective/office/tv/screen/duolingo/utils/FlagMapper.kt:1]: it is beautiful

File: tv-app/effecticeTV/app/src/main/java/band/effective/office/tv/ui/theme/Theme.kt [hunk 4]
```diff
@@ -4,7 +4,11 @@ import androidx.compose.material.MaterialTheme
 import androidx.compose.material.darkColors
 import androidx.compose.runtime.Composable
 import androidx.compose.ui.graphics.Color
-import com.example.effecticetv.ui.theme.*
+import band.effective.office.tv.ui.theme.Shapes
+import com.example.effecticetv.ui.theme.BlackOlive
+import com.example.effecticetv.ui.theme.CharlestonGreen
+import com.example.effecticetv.ui.theme.QuickSilver
+import com.example.effecticetv.ui.theme.VividTangelo
```
Comment (UserNameMax) [inline tv-app/effecticeTV/app/src/main/java/band/effective/office/tv/ui/theme/Theme.kt:11]: com.example?

**Review by UserNameMax — APPROVED** [review]

File: tv-app/effecticeTV/app/src/main/java/band/effective/office/tv/core/ui/screen_with_controls/TimerSlideShow.kt
Comment (UserNameMax) [reply tv-app/effecticeTV/app/src/main/java/band/effective/office/tv/core/ui/screen_with_controls/TimerSlideShow.kt:1]: ok

File: tv-app/effecticeTV/app/src/main/java/band/effective/office/tv/screen/duolingo/utils/FlagMapper.kt
Comment (gull192) [reply tv-app/effecticeTV/app/src/main/java/band/effective/office/tv/screen/duolingo/utils/FlagMapper.kt:1]: yes, we will have to do something with these flags

**Pull request merged**